src/routes/AppRouter.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { Route, BrowserRouter as Router, Switch } from 'react-router-dom';
-import ErrorPage from '../pages/ErrorPage';
-import HomePage from '../pages/HomePage';
+import ErrorPage from '../pages/ErrorPage/ErrorPage';
+import HomePage from '../pages/HomePage/HomePage';
```

Comment:
The previous import format is more correct. If you add `export * from './ErrorPage` to an `index.js` file inside the component folder, you don't need to import twice. `import ErrorMessage from '../pages/HomePage ` is enough.

Reply:
Also let's avoid using `export default` to take full advantage of ES6 named imports to reduce imports. Please follow this resources.
[Link 1](https://blog.bitsrc.io/structuring-a-react-project-a-definitive-guide-ac9a754df5eb)
[Link 2](https://stackoverflow.com/questions/29722270/is-it-possible-to-import-modules-from-all-files-in-a-directory-using-a-wildcard/31402500#31402500)

Reply:
I agree